tests/api/runtimes/test_kubejob.py

```diff
@@ -275,7 +275,12 @@ def test_run_with_priority_class_name(self, db: Session, k8s_secrets_mock):
                 {"zone": "us-east", "gpu": "false"},
             ),
             # Common and user node selectors provided
-            ({"zone": "us-east"}, {}, {"test": "user"}, {"test": "user"}),
+            (
+                {"zone": "us-east"},
+                {},
+                {"test": "user"},
+                {"zone": "us-east", "test": "user"},
```

Comment:
what would I do if the global default node selector is `{"test": "user"}` and I only want `{"zone": "us-east"}`?

Reply:
then on your function node selector you can do: {"test": "", "zone": "us-east"}, and because the function takes precedence it will be the resulted node selector for the run. 
(empty labels are being removed when creating the pods)